cypress/integration/project_creation.spec.js

```diff
@@ -1,23 +1,29 @@
 context("project creation", () => {
   context("project creation modal", () => {
     beforeEach(() => {
-      cy.visit("./public/index.html#/projects");
+      cy.visit("./public/index.html#/profile");
     });
 
     // TODO(rudolfs): test empty project listing has wording and button
 
-    it("can be opened via the projects page and closed by pressing cancel", () => {
-      cy.get('[data-cy="new-project-button"]').click();
+    it("can be opened via the profile context menu and closed by pressing cancel", () => {
+      cy.get('[data-cy="profile-context-menu"]').click();
+      cy.get('[data-cy="dropdown-menu"] [data-cy="new-project"]').click({
+        force: true
```

Comment:
Yes. I had to add it because we have a bug in the dropdown, which will be addressed in: https://github.com/radicle-dev/radicle-upstream/issues/246. Added a note in that issue to also fix the specs.